source/docs/3/digitizing_basics.rst

```diff
@@ -40,287 +40,240 @@ Data Source [LINZ]_
 
-8. When creating a GIS layer, you must decide on the attributes that each feature will have. Since this is a roads layer, we additionally will have 2 basic attributes - Name and Class. In :guilabel:`New Field` Enter ``Name`` of the type :guilabel:`Text data`, with ``50`` as :guilabel:`Maximum length` and click :guilabel:`Add to attribute list.` Now create a new attribute ``Class`` of the type :guilabel:`Text data`, with ``50`` as :guilabel:`Maximum length`. Click :guilabel:`OK`
+ .. image:: /static/3/digitizing_basics/images/04.png
+    :align: center   
  
-
-.. image:: /static/3/digitizing_basics/images/8.png
-   :align: center
-
-
-9. Once the ``Roads`` layer is loaded, click the :guilabel:`Toggle Editing` button to put the layer in editing mode.
-
-.. image:: /static/3/digitizing_basics/images/9.png
-   :align: center
-
-10. Click the :guilabel:`Add Line Feature` button. Click on the map canvas to add a
-    new vertex. Add new vertices along with the road feature. Once you have digitized
-    a road segment, right-click to end the feature.
-
+05. Once the process finishes, the dialog box will show the pyramids without a cross. This indicates the Image Pyramids build is completed. Click :guilabel:`OK`.
+
+ .. image:: /static/3/digitizing_basics/images/05.png
+    :align: center
+   
+06. Before we start, we need to set default **Digitizing Options**. Go to :menuselection:`Settings --> Options...`.
+
+ .. image:: /static/3/digitizing_basics/images/06.png
+    :align: center
+	
+07. Select the :guilabel:`Digitizing` tab in the :guilabel:`Options` dialog. Check the :guilabel:`Enable snapping by default`  under :guilabel:`Snapping` section. In :guilabel:`Default snap mode` choose :guilabel:`Vertex`. This will allow you to snap to the nearest vertex. I also prefer to set the :guilabel:`Default snapping tolerance` and :guilabel:`Search radius for vertex edits` in pixels instead of map units. This will ensure that the snapping distance remains constant regardless of zoom level. Depending on your computer screen resolution, you may choose an appropriate value. Click :guilabel:`OK`.
+
+ .. image:: /static/3/digitizing_basics/images/07.png
+    :align: center
+   
+08. Now we are ready to start digitizing. We will first create a road layer and digitize the roads around the park area. Select :guilabel:`New GeoPackage Layer...` icon from Panels. A GeoPackage is an open, non-proprietary, platform-independent, and standards-based data format for a geographic information system implemented as an SQLite database container. This makes it much easier to move it around instead of a bunch of shapefiles. In this tutorial, we create a couple of polygon layers and a line layer so that a GeoPackage will be better suited. You can always load a GeoPackage and export layers as a shapefile or any other format you want.

```

Comment:
Show menu navigation (not toolbar button)

Reply:
Done.